# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../zaptel/split_make_install.patch
# Copyright (C) 2006 The OpenSDE Project
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Just build and install userspace stuff of zaptel 

--- ./Makefile.orig	2006-09-11 06:49:22.000000000 -0400
+++ ./Makefile	2006-09-11 10:39:02.000000000 -0400
@@ -137,6 +137,8 @@
 
 all: $(BUILDVER) $(LIBTONEZONE_SO)
 
+bins: $(BINS)
+
 linux24: prereq $(MODULESO) wct4xxp/wct4xxp.o $(BINS)
 
 linux26: prereq $(BINS)
@@ -315,14 +317,7 @@
 	tar -zxf mISDNuser.tar.gz
 	$(MAKE) -C mISDNuser install
 
-install: all devices firmware
-	install -D -m 755 ztcfg $(INSTALL_PREFIX)/sbin/ztcfg
-	if [ -f sethdlc-new ]; then \
-		install -D -m 755 sethdlc-new $(INSTALL_PREFIX)/sbin/sethdlc; \
-	elif [ -f sethdlc ]; then \
-		install -D -m 755 sethdlc $(INSTALL_PREFIX)/sbin/sethdlc ; \
-	fi
-	if [ -f zttool ]; then install -D -m 755 zttool $(INSTALL_PREFIX)/sbin/zttool; fi
+install-modules:
 ifeq ($(BUILDVER),linux26)
 	for x in $(MODULESKO); do \
 		rm -f $(INSTALL_PREFIX)/lib/modules/$(KVERS)/extra/$$x ; \
@@ -337,6 +332,14 @@
 		rm -f $(INSTALL_PREFIX)/lib/modules/$(KVERS)/misc/wcfxsusb.o; \
 	fi; \
 	rm -f $(INSTALL_PREFIX)/lib/modules/$(KVERS)/misc/wcfxs.o
+install: 
+	install -D -m 755 ztcfg $(INSTALL_PREFIX)/sbin/ztcfg
+	if [ -f sethdlc-new ]; then \
+		install -D -m 755 sethdlc-new $(INSTALL_PREFIX)/sbin/sethdlc; \
+	elif [ -f sethdlc ]; then \
+		install -D -m 755 sethdlc $(INSTALL_PREFIX)/sbin/sethdlc ; \
+	fi
+	if [ -f zttool ]; then install -D -m 755 zttool $(INSTALL_PREFIX)/sbin/zttool; fi	
 	install -D -m 755 $(LIBTONEZONE_SO) $(INSTALL_PREFIX)/usr/lib/$(LIBTONEZONE_SO).$(LIBTONEZONE_SO_MAJOR_VER).$(LIBTONEZONE_SO_MINOR_VER)
 	[ `id -u` = 0 ] && /sbin/ldconfig || :
 	rm -f $(INSTALL_PREFIX)/usr/lib/$(LIBTONEZONE_SO)
